src/drivers/visitor.rs

```diff
@@ -0,0 +1,81 @@
+use std::io::Read;
```

Comment:
Just file organisation wise, I think I'd prefer something like

```
drivers.rs -- contains just `mod`s and re-exports maybe
drivers/crates_io.rs -- the `fetch_crate_cratesio` function
drivers/the_other_one(s).rs

drivers/crates_io/vfs.rs
drivers/crates_io/visitor.rs
```

Reply:
Ok, but note that this would make the diff a lot bigger.

Reply:
Done!